Tools/TestWebKitAPI/Tests/IPC/IPCSerialization.mm

```diff
@@ -533,6 +533,19 @@ static bool wkNSURLProtectionSpace_isEqual(NSURLProtectionSpace *a, SEL, NSURLPr
     return true;
 }
 
+static bool CNPostalAddressTesting_isEqual(CNPostalAddress *a, CNPostalAddress *b)
```

Comment:
`#if HAVE(CONTACTS)`?

Reply:
Updating to protect it with "has passkit and !watchOS" like the rest of the contacts related code.